subsys/bluetooth/controller/Kconfig

```diff
@@ -62,6 +62,7 @@ if BT_CTLR
 
 choice BT_LL_CHOICE
 	prompt "Bluetooth Link Layer Selection"
+	default BT_LL_SW_SPLIT
```

Comment:
BT_LL_SW_SPLIT is the first choice in the list below, an explicit default is unnecessary.

Reply:
I believe the patch that you reviewed here was accidentally added to this PR.

@tejlmand : Is it possible to just include the commit in question in this PR?

Reply:
this was due to a push to upstream instead of downstream remote when syncing two PR's.
Has been removed again.

Sorry for the noise.